core/src/main/scala/kafka/log/LogConfig.scala

```diff
@@ -124,8 +124,8 @@ object LogConfig {
   val MessageFormatVersionProp = "message.format.version"
   val MessageTimestampTypeProp = "message.timestamp.type"
   val MessageTimestampDifferenceMaxMsProp = "message.timestamp.difference.max.ms"
-  val LeaderThrottledReplicasListProp = "quota.leader.replication.throttled.replicas"
-  val FollowerThrottledReplicasListProp = "quota.follower.replication.throttled.replicas"
+  val LeaderThrottledReplicasListProp = "leader.replication.throttled.replicas"
+  val FollowerThrottledReplicasListProp = "follower.replication.throttled.replicas"
```

Comment:
What's the motivation for this change, make the names shorter? Oh, probably to make them consistent with the rate config names?